### PR TITLE
Alerts documentation

### PR DESCRIPTION
**Pull request by pd0wm**

### CODE REVIEW

File: selfdrive/controls/lib/events.py [hunk 1]
```diff
@@ -737,15 +806,24 @@ def startup_fuzzy_fingerprint_alert(CP: car.CarParams, sm: messaging.SubMaster,
     ET.NO_ENTRY: NoEntryAlert("Reverse Gear"),
   },
 
+  # On cars that use stock ACC the car can decide to cancel ACC for various reasons.
+  # When this happens we can no long control the car so the user needs to be warned immediately.
```
Comment (VirtuallyChris) [inline selfdrive/controls/lib/events.py:810]: Can this also trigger when the 'main' switch is pressed on nidec and bosch?

File: selfdrive/controls/lib/events.py [hunk 2]
```diff
@@ -479,18 +515,24 @@ def startup_fuzzy_fingerprint_alert(CP: car.CarParams, sm: messaging.SubMaster,
...
   EventName.fanMalfunction: {
     ET.PERMANENT: NormalPermanentAlert("Fan Malfunction", "Contact Support"),
   },
 
+  # Camera is not outputting frames at a constant framerate
   EventName.cameraMalfunction: {
     ET.PERMANENT: NormalPermanentAlert("Camera Malfunction", "Contact Support"),
   },
 
+  # Unused
```
[the remark this answers is not in the document]
Comment (pd0wm) [reply selfdrive/controls/lib/events.py:528]: This was throwing false positive alerts a while back when the ublox module was overloaded by too many satellites. We should probably bring it back in some other form.

File: selfdrive/controls/lib/events.py [hunk 3]
```diff
@@ -596,6 +638,10 @@ def startup_fuzzy_fingerprint_alert(CP: car.CarParams, sm: messaging.SubMaster,
     ET.NO_ENTRY: NoEntryAlert("Gear not D"),
   },
 
+  # This alert is thrown when the calibration angles are outside of the acceptable range.
+  # For example if the device is pointed too much to the left or the right.
+  # Usually this can only be solved by removing the mount from the windshield completely,
+  # and attaching while making sure the device is pointed straight forward and is level.
```
Comment (pd0wm) [inline selfdrive/controls/lib/events.py:644]:
```suggestion
  # and attaching while making sure the device is pointed straight forward and is level.
  # See https://comma.ai/setup for more information
```

File: selfdrive/controls/lib/events.py [hunk 4]
```diff
@@ -691,12 +753,20 @@ def startup_fuzzy_fingerprint_alert(CP: car.CarParams, sm: messaging.SubMaster,
...
+  # causing the connection to the panda to be lost
   EventName.usbError: {
     ET.SOFT_DISABLE: SoftDisableAlert("USB Error: Reboot Your Device"),
     ET.PERMANENT: NormalPermanentAlert("USB Error: Reboot Your Device", ""),
     ET.NO_ENTRY: NoEntryAlert("USB Error: Reboot Your Device"),
   },
 
+  # This alert can be thrown for the following reasons:
+  # - No CAN data received at all
+  # - CAN data is received, but some messages don't have valid checksums
```
Comment (pd0wm) [inline selfdrive/controls/lib/events.py:766]:
```suggestion
```

File: selfdrive/controls/lib/events.py [hunk 5]
```diff
@@ -691,12 +753,20 @@ def startup_fuzzy_fingerprint_alert(CP: car.CarParams, sm: messaging.SubMaster,
...
+  # - CAN data is received, but some message have the wrong counter
```
Comment (pd0wm) [inline selfdrive/controls/lib/events.py:767]:
```suggestion
```

**Pull request merged**